### PR TITLE
Typo fix: te_estimator -> po_estimator.

### PR DESCRIPTION
**Pull request by kamilest**

### CODE REVIEW

File: catenets/models/torch/utils/model_utils.py [hunk 1]
```diff
@@ -81,10 +80,10 @@ def train_wrapper(
     **kwargs: Any,
 ) -> None:
     if hasattr(estimator, "train"):
-        log.debug(f"Train PyTorch network {estimator}")
+        print(f"Train PyTorch network {estimator}")
```
Comment (AliciaCurth) [inline catenets/models/torch/utils/model_utils.py:83]: What is the reason for this change from log to print?

Comment (kamilest) [reply catenets/models/torch/utils/model_utils.py:83]: Hmm yeah I think I accidentally left a commit that shouldn't be there... I wonder can this commit be reverted but the other kept as it is...

Comment (bcebere) [reply catenets/models/torch/utils/model_utils.py:83]: `git revert <commit hash>`

commit hash can be found in `git log`.

**Comment by AliciaCurth** [comment]
Fixed in Bogdan's last PR :)
